server/registry/actions_deployment_revisions_test.go

```diff
@@ -162,6 +162,51 @@ func TestTagApiDeploymentRevision(t *testing.T) {
 	})
 }
 
+func TestTagApiDeploymentRevisionResponseCodes(t *testing.T) {
+	tests := []struct {
+		desc string
+		tag  string
+		want codes.Code
+	}{
+		{
+			desc: "contains uppercase leters",
+			tag:  "TestTag",
+			want: codes.InvalidArgument,
+		},
+		{
+			desc: "single dash",
+			tag:  "-",
+			want: codes.InvalidArgument,
+		},
+		{
+			desc: "valid one-character tag",
+			tag:  "x",
+			want: codes.OK,
+		},
+		{
+			desc: "valid tag",
+			tag:  "latest",
+			want: codes.OK,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.desc, func(t *testing.T) {
+			ctx := context.Background()
+			server := defaultTestServer(t)
+			if err := seeder.SeedDeployments(ctx, server, &rpc.ApiDeployment{Name: "projects/my-project/locations/global/apis/a/deployments/d"}); err != nil {
+				t.Fatalf("Setup/Seeding: Failed to seed registry: %s", err)
+			}
+			if _, err := server.TagApiDeploymentRevision(ctx, &rpc.TagApiDeploymentRevisionRequest{
+				Name: "projects/my-project/locations/global/apis/a/deployments/d",
```

Comment:
@theganyo I noticed we have an open issue #601 about implicit tagging of the latest revision - which I believe is what is happening here (successfully)

Reply:
Yes, and I just looked through the code to verify: The changes I made [in this PR](https://github.com/apigee/registry/pull/826) allow revision requests to use a non-revision name which will assume the latest revision.